src/core.js

```diff
@@ -429,18 +429,18 @@ extend( QUnit, {
 
 // `onErrorFnPrev` initialized at top of scope
 // Preserve other handlers
-onErrorFnPrev = window.onerror;
+onErrorFnPrev = global.onerror;
```

Comment:
Same as above might apply here - do we really expect `onerror` to be available outside of any browser?